ykrt/src/compile/jitc_yk/codegen/x64/mod.rs

```diff
@@ -771,6 +854,8 @@ impl<'a> Assemble<'a> {
                 inlined_frames: gd.inlined_frames,
                 guard: Guard::new(),
             });
+
+            stack_sizes.push(self.ra.stack_size());
```

Comment:
It probably doesn't make a difference, but if we didn't want to use a vec here we could just move the `max_stack_size` calculation in here, so do it after every guard, and then just return an integer here.

Reply:
Fixed in 4efbf3d9.